app/mailers/notifications_mailer.rb

```diff
@@ -0,0 +1,7 @@
+class NotificationsMailer < ActionMailer::Base
+  def pending_cards(user)
+    @user = user
+    @url = ENV['URL_DEFAULT']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.